src/Web/Web.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>Contoso.FraudProtection.Web</RootNamespace>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="BuildBundlerMinifier" Version="3.2.449" />
-    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="3.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="5.0.0" />
```

Comment:
just curious, why isn't this using version `5.0.10` similar to `Microsoft.EntityFrameworkCore.InMemory` and `Microsoft.EntityFrameworkCore.Tools`

Reply:
No particular reason, this was the first combination of versions that built successfully/worked without any errors.